src/Toolkit/Toolkit/JobManager/JobManager.iOS.cs

```diff
@@ -0,0 +1,213 @@
+        }
+
+        /// <summary>
+        /// Ends any current background task.
+        /// </summary>
+        private void EndCurrentBackgroundTask()
+        {
+            if (_backgroundTaskIdentifier.HasValue)
+            {
+                UIApplication.SharedApplication.EndBackgroundTask(_backgroundTaskIdentifier.Value);
+                _backgroundTaskIdentifier = null;
+            }
+        }
+
+        private void BeginBackgroundTask()
+        {
+            // Jobs that are started but do not yet have a server job ID are jobs that
+            // can benefit from starting a background task for extra execution time.
+            // This will hopefully allow a job to get to the polling state before the app is suspended.
+            // Once in a polling state that's where background refreshes can check job status.
+            if (!Jobs.Any(j => j.Status == JobStatus.Started && string.IsNullOrEmpty(j.ServerJobId))) return;
+            // Already started.
+            if (_backgroundTaskIdentifier.HasValue) return;
+
+            _backgroundTaskIdentifier = UIApplication.SharedApplication.BeginBackgroundTask(() =>
+            {
+                EndCurrentBackgroundTask();
+            });
+        }
+
+        internal bool ValidateInfoPlist()
+        {
+            var bgtask = Foundation.NSBundle.MainBundle.ObjectForInfoDictionary("BGTaskSchedulerPermittedIdentifiers");
+            if (bgtask is null)
+            {
+                //throw new InvalidOperationException("Missing 'BGTaskSchedulerPermittedIdentifiers' entry in info.plist");
+                System.Diagnostics.Trace.WriteLine("Missing 'BGTaskSchedulerPermittedIdentifiers' entry in info.plist", "ArcGIS Toolkit");
+                return false;
+            }
+            // Get the array of permitted identifiers from the info.plist BGTaskSchedulerPermittedIdentifiers
+            // and ensure that `com.esri.ArcGISToolkit.jobManager.statusCheck` is present.
+            var array = bgtask as Foundation.NSArray;
+            if (array == null)
+            {
+                System.Diagnostics.Trace.WriteLine("'BGTaskSchedulerPermittedIdentifiers' entry in info.plist is not an array.", "ArcGIS Toolkit");
+                //throw new InvalidOperationException("'BGTaskSchedulerPermittedIdentifiers' entry in info.plist is not an array.");
+                return false;
+            }
+            for (nuint i = 0; i < array.Count; i++)
+            {
+                var value = array.GetItem<Foundation.NSString>(i);
+                if (value != null && value.ToString() == DefaultsKey)
+                {
+                    return true;
+                }
+            }
+            System.Diagnostics.Trace.WriteLine($"'BGTaskSchedulerPermittedIdentifiers' must contain '{DefaultsKey}'.", "ArcGIS Toolkit");
+            return false;
+            throw new InvalidOperationException($"'BGTaskSchedulerPermittedIdentifiers' must contain '{DefaultsKey}'.");
```

Comment:
Were you undecided about returning or throwing? The xmldoc on PreferredBackgroundStatusCheckSchedule says that we throw, but the setter actually just fails.